src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+      class="main-form"
+    >
+
+      <fieldset class="main-form__fieldset">
+        <legend>Personal Information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="how-old">How old are You?</label>
+          <input
+            type="number"
+            name="old"
+            id="how-old"
+            value="12"
+            min="12"
+            max="100"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="date-of-birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="date"
+            id="date-of-birth"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="term-of-agree">I accept the lhe term of the agreement</label>
+          <input
+            type="checkbox"
+            name="term-of-agree"
```

Comment:
4th item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), name attributes should be in camelCase, make sure to fix it everywhere
```suggestion
            name="termOfAgree"
```